dev/ProgressRing/ProgressRing.cpp

```diff
@@ -305,8 +305,17 @@ void ProgressRing::UpdateLottieProgress()
         const double range = Maximum() - min;
         const double fromProgress = (m_oldValue - min) / range;
         const double toProgress = (value - min) / range;
+        if (fromProgress < toProgress)
+        {
+            const auto _ = player.PlayAsync(fromProgress, toProgress, false);
+        }
+        else
+        {
+            player.PlaybackRate(-1);
+            const auto _ = player.PlayAsync(fromProgress, toProgress, false);
```

Comment:
How is this doing a jump backwards ? I would have expected it to animate backwards (perhaps that is broken). I think SetProgress might be the one that does the jump.

Reply:
I think @teaP said that the animation does not support a negative playback rate. Because of that, it probably directly jumps to the new progress. Should I switch to SetProgress instead of trying to animate backwards?

Reply:
Yes please. This might otherwise change in behavior if we fix that.

Reply:
Maybe for customization purposes, it is better to go with animating backwards in case someone wants to switch out the animated visual and also wants to be able to animate backwards. I am not sure if users would be able to have it animate backwards easily if we just set the progress of the animation instead of animating.

Reply:
Update it to use SetProgress now since we shouldn't risk changing the behavior because of a bugfix in another area.